odxtools/cli/_print_utils.py

```diff
@@ -154,15 +154,15 @@ def extract_parameter_tabulation_data(parameters: List[Parameter]) -> Table:
     table.add_column("Value Type", justify="left", style="white")
     table.add_column("Linked DOP", justify="left", style="white")
 
-    name = []
-    byte = []
-    bit_length: List[Optional[int]] = []
-    semantic = []
-    param_type = []
-    value: List[Optional[str]] = []
-    value_type: List[Optional[str]] = []
-    data_type: List[Optional[str]] = []
-    dop: List[Optional[str]] = []
+    name: List[Optional[object]] = []
```

Comment:
I defined it as object to solve the cli issue. So you say I keep it simply like name = [ ]?

Reply:
I am just proposing a simpler syntax, yours work if you want to keep it

Reply:
okay sure thank you.

Reply:
AFAICS this code is still unchanged? But actually I prefer to use `List[Any]` here, or -- even better -- replacing the `Any` by the types which can actually occur here. Since the latter might be quite a bit of work, `Any` is okay for now...

Reply:
Yeah. you are right. let me fix this.

Reply:
@andlaus I have made changes, I kept the Any for two variables because here I was getting static type error if defined as int, str or None.